packages/react/src/components/multi-factor-auth/multi-factor-auth.tsx

```diff
@@ -1,20 +1,32 @@
 import type { Factor } from "@slashid/slashid";
+import type { FormProps } from "../form";
 import { Form } from "../form";
 import { LoggedIn } from "../logged-in";
 import { LoggedOut } from "../logged-out";
 import { MFAProvider } from "../../context/config-context";
 import { TextConfig } from "../text/constants";
 
-type MFAProps = {
+type MultiFactorAuthProps = FormProps & {
   factors: Factor[];
   text?: Partial<TextConfig>;
 };
 
-export function MFA({ factors, text }: MFAProps) {
+/**
+ * First-class Multi-Factor Authentication experience.
+ * This component can be used instead of regular `<Form />`.
+ * After successful authentication with one of the factors defined in a `<ConfigurationProvider>`,
+ * the user will _immediately_ be prompted with second step - one of the `factors`.
+ */
+export function MultiFactorAuth({
```

Comment:
`Form` should receive an extended set of props in order for developers to be able to override the default values set in the `ConfigurationProvider`. These would be:
- `factors`
- `text`

These values would be optional and if set merged with the `ConfigurationProvider` values.

Then the MultiFactorAuth component could receive an array of steps, each of them having a config entry consisting of the above values.

Reply:
We can override `useConfiguration` to apply overrides to the config provider values.